content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+
+* A Trusted Platform Module (TPM) security chip to handle public and private keys. Most recent business-grade laptops, desktops, and smartphones have TPMs.
+* A camera or biometric reader that supports the types of input required; for example, to scan a fingerprint, a fingerprint reader is required. Note that a biometric sensor is not required for a platform authenticator. The point is that the authenticator must support the type of input required, which could just as well be PIN for user verification, or indeed, in the case where there is no user verification at all, the TPM would be used as a U2F style second factor authenticator.
+
+Roaming authenticators, also known as cross-platform or external authenticators, such as a hardware security key. These authenticators can be used with a laptop or mobile device, etc., and are thus cross-platform. This characteristic enables their use to establish a secure source for verifying the user’s identity (the "root of trust") and for delegating trust to specific devices in the user’s control. The user can therefore authorize other devices as needed ("bootstrap" them). Should one of those other devices be lost or become corrupted, the roaming authenticator is used to authorize a new device. For these reasons, it is a roaming authenticator such as a security key that should be the primary authenticator.
+
+==== What is a trusted device?
+In the realm of traditional authentication the concept of Trusted Devices was introduced as a form of 2FA to make the login experience easier for a user who frequents a device. This often included the storage of a device identifier as a cookie, in cache, or in local storage. This presents an attack avenue, if an attacker gains access to a user's computer, or finds a way to spoof the machine, then all they require is a compromised password to gain access to a user's account. A Platform Authenticator can act as a Trusted Device by relying on the WebAuthn credential already built into the device. For an attacker it’s not just as easy as having access to the users device, they will also need knowledge of the users PIN, or access to the users biometrics (Fingerprint for Android Biometrics, Face for Face ID, etc.)
+
+==== Hardware backed vs copyable credentials
+Some technology providers have begun to implement copyable credentials within their operating systems/ecosystems, oftentimes this is branded as passkeys. You should be aware of the difference between the two credential types so that you can guide your authentication strategy.
+
+Hardware backed credentials are credentials where the private key never leaves the piece of hardware that they were originally created on. This greatly minimizes the risk of your credential being exposed to attackers as they would need the physical device in order to access any of your accounts. These credentials allow for the highest degree of security.
```

Comment:
Consider working in the keyword `high assurance`

Reply:
Added in newest commit